server/etcdmain/config.go

```diff
@@ -286,6 +286,7 @@ func newConfig() *config {
 	fs.DurationVar(&cfg.ec.ExperimentalWatchProgressNotifyInterval, "experimental-watch-progress-notify-interval", cfg.ec.ExperimentalWatchProgressNotifyInterval, "Duration of periodic watch progress notifications.")
 	fs.DurationVar(&cfg.ec.ExperimentalDowngradeCheckTime, "experimental-downgrade-check-time", cfg.ec.ExperimentalDowngradeCheckTime, "Duration of time between two downgrade status check.")
 	fs.DurationVar(&cfg.ec.ExperimentalWarningApplyDuration, "experimental-warning-apply-duration", cfg.ec.ExperimentalWarningApplyDuration, "Time duration after which a warning is generated if request takes more time.")
+	fs.DurationVar(&cfg.ec.ExperimentalWarningUnaryRequestDuration, "experimental-warning-unary-request-duration", cfg.ec.ExperimentalWarningUnaryRequestDuration, "Time duration after which a warning is generated if a unary request takes more time.")
```

Comment:
Lets add a mention of the default value here as well. Maybe we can print the default value from a variable or this way:

```suggestion
	fs.DurationVar(&cfg.ec.ExperimentalWarningUnaryRequestDuration, "experimental-warning-unary-request-duration", cfg.ec.ExperimentalWarningUnaryRequestDuration, "Time duration after which a warning is generated if a unary request takes more time. Default value is 300ms.")
```

Reply:
Hi @lilic , thanks for the review. 
No problem, I can add the default value to the explanation, however, the flag help message is comming from the `help.go` file and it does contain the default value.
```  
--experimental-warning-unary-request-duration '300ms'
    Warning is generated if a unary request takes  more than this duration.
```
Futhermore, most of other flag arguments in  `config.go` do not mention their  default values either.
What do you think ?

Reply:
Yes sounds good to me! Added one suggestion for that.